### PR TITLE
Minor refactoring

### PR DESCRIPTION
**Pull request by smalleni**
Signed-off-by: Sai Sindhur Malleni <smalleni@redhat.com>

### CODE REVIEW

**Comment by smalleni** [comment]
@dry923 this is tested locally and ready for review. Improves some logging, moves around some code to make it cleaner.

File: snafu/log_generator_wrapper/trigger_log_generator.py [hunk 1]
```diff
@@ -276,15 +282,17 @@ def emit_actions(self):
...
-                    logger.info(
-                        "Current messages received is {}, waiting more time for kafka".format(
-                            messages_received
-                        )
-                    )
                 if messages_received == message_count:
                     received_all_messages = True
                 else:
                     logger.info("Message check failed. Retrying until timeout")
+                    if self.kafka_bootstrap_server:
```
Comment (dry923) [inline snafu/log_generator_wrapper/trigger_log_generator.py:289]: This if is redundant with the if at line 274. It wouldn't fall into here if the bootstrap server was undefined

File: snafu/log_generator_wrapper/log_generator_wrapper.py [hunk 2]
```diff
@@ -59,6 +59,7 @@ def __init__(self, parent_parser):
         )
         parser.add_argument("--kafka-bootstrap-server", help="The Kafka Service to connect to")
         parser.add_argument("--kafka-topic", help="The Kafka topic to verify logs")
+        parser.add_argument("--kafka-check", help="Verify if logs made it to kafka sink", action="store_true")
```
Comment (dry923) [inline snafu/log_generator_wrapper/log_generator_wrapper.py:62]: @smalleni Curious why you are adding a verification true|false. If you are providing the topic/bootstrap then the verification would be implied. Or is there another time/reason you would pass that info but not want to verify?

File: snafu/log_generator_wrapper/trigger_log_generator.py (hunk 1 above)
Comment (smalleni) [reply snafu/log_generator_wrapper/trigger_log_generator.py:289]: It is not, I only want to execute L290-L295 for kafka. Line 274 is an OR condition, so we enter into it for any condition that meets it (ES, cloudwatch or kafka), however I now only want to check for kafka.

Comment (dry923) [reply snafu/log_generator_wrapper/trigger_log_generator.py:289]: Ahhhhhhhhh ok. nevermind.

File: snafu/log_generator_wrapper/log_generator_wrapper.py (hunk 2 above)
Comment (dry923) [reply snafu/log_generator_wrapper/log_generator_wrapper.py:62]: That's fair.  I just wanted to confirm the intent really.

**Review by jtaleric — APPROVED** [review]
lgtm

**Pull request merged**